README.md

```diff
@@ -10,15 +10,19 @@ To install the form-generator library, you can use: npm install form-generator-t
 ### Install Form Generator One Step
 
 ```ts
+import { propertiesConfig } from "../node_modules/form-generator-typescript/src/config/config-attributes";
+import { translationsEn } from "../node_modules/form-generator-typescript/src/config/config-en";
 import { FormGeneratorOneStep } from "../node_modules/form-generator-typescript/src/form-generator/form-generator-one-step";
 import { FormHandlerOneStep } from "../node_modules/form-generator-typescript/src/form-generator/form-handler-one-step";
 import { FormStylesOneStep } from "../node_modules/form-generator-typescript/src/form-styles/form-styles-one-step";
 
-const formGenerator = new FormGeneratorOneStep(new FormHandlerOneStep());
+const config = propertiesConfig(translationsEn);
+const apiKeyNumverify = 'YOUR_GENERATED_API_KEY';
+const formGenerator = new FormGeneratorOneStep(new FormHandlerOneStep(config));
 const formContainer = document.getElementById('form-container');
 new FormStylesOneStep();
 
-formGenerator.generateForm().then((form) => {
+formGenerator.generateForm(apiKeyNumverify).then((form) => {
```

Comment:
a jak przekazesz config ? albo jak ktos sobie zrobi swoj config ?

Reply:
const config = propertiesConfig(translationsEn);
const formGenerator = new FormGeneratorOneStep(new FormHandlerOneStep(config));

tutaj przekazuje config a zeby wykorzystac inny to podmienialam propertiesConfig na nowy config